### DISCUSSION TITLE
Explorer patcher crashed more than three times

### DISCUSSION THREAD

**Discussion by vandelin586** [valinet/ExplorerPatcher#2586] [General]
Guessing a recent update to my win 11 pro 23H2 . I have to uninstall explorer patcher to get even "my computer" to open, when opening any file explore window, the whole desktop flashes and it never opens. Sorry dont know what other details are needed. I'll check in on this discussion soon

**Comment by vandelin586** [comment]
Application: Explorer.EXE
Framework Version: v4.0.30319
Description: The process was terminated due to an unhandled exception.
Exception Info: exception code c0000005, exception address 00007FFBD03A862B

Faulting application name: Explorer.EXE, version: 10.0.22621.2915, time stamp: 0x7b3b7801
Faulting module name: ExplorerFrame.dll, version: 10.0.22621.2915, time stamp: 0x23ad12f5
Exception code: 0xc0000005
Fault offset: 0x000000000000862b
Faulting process id: 0x0x190C
Faulting application start time: 0x0x1DA2F0DB07965EF
Faulting application path: C:\WINDOWS\Explorer.EXE
Faulting module path: C:\WINDOWS\SYSTEM32\ExplorerFrame.dll
Report Id: 99721f86-9fc6-4a8e-ae60-bf924b561281
Faulting package full name: 
Faulting package-relative application ID:

**Comment by apjarnold** [comment]
> Guessing a recent update to my win 11 pro 23H2 . I have to uninstall explorer patcher to get even my computer to open, when opening any file explore window, the whole desktop flashes and it never opens. Sorry dont know what other details are needed. I'll check in on this discussion soon

Same here after windows update, file explorer would flash and not open, uninstalled Explorer Patcher all working fine.

Edition	Windows 11 Pro
Version	23H2
Installed on	‎22/‎09/‎2023
OS build	22635.2915
Experience	Windows Feature Experience Pack 1000.22682.1000.0

**Comment by pyrates999** [comment]
Did uninstalling EP and then installing EP fix it?

**Reply by vandelin586** [reply]
I did not have luck but I believe under issues is a similar issue being discussed which i believe is the same as mine .... within explorerpatcher menu options you can select beta type version updates? should I select this?

**Reply by vandelin586** [reply]
Reinstalled newest version and it is working ... Thank you !!!

**Reply by pyrates999** [reply]
Please close this then.

**Comment by Amrsatrio** [comment]
Please install the latest pre-release.

We strive to make EP functional on beta builds, but we're sorry for delays in fixes like these.

**Reply by apjarnold** [reply]
I will give it a try tomorrow and report back.

**Reply by apjarnold** [reply]
Could not wait , reinstalled and working fine. Thank you.

**Reply by pyrates999** [reply]
Please close this then.

**Reply by vandelin586** [reply]
Reinstall of newest version worked for me .... thank you so much

**Comment by influential-eliot** [comment]
Hiya, I noticed before that there was conversation about this incident frequently occurring with Windows Insider builds.

So I have waited until I was *finally* off that before getting to EP, as I'm so done with a bottom task bar and more.

However, I was afraid of crash loops like this, so I'm purely asking here (_rather than starting a new topic to pile on_) as to current status on this kind of thing.

Anyway, I know this is all free software, so this is not a complaint ... but I just wondered that if I install 23H2 then this, will things be OK?

**Reply by Amrsatrio** [reply]
I've tested EP up to the latest Dev build. Don't worry, you can go ahead and install 23H2 and keep using EP because 23H2 is basically 23H2 with differentiated build number and longer support term.

Although, if you use the old taskbar, I recommend to stay (force your system) on 22H2 because the update that nukes certain parts of the old taskbar is around the corner.

**Reply by influential-eliot** [reply]
Thanks, @Amrsatrio, much obliged.

Could you just confirm what you mean by "_old taskbar_"?

Only I intend to use EP to move the taskbar to the left or right side of the screen, with the corner start button, and menu opening on the currently active screen for the mouse. Does that constitute "_old taskbar_" or do you mean something else?

**Reply by Amrsatrio** [reply]
That is the Windows 10 taskbar, which I referred to as "old taskbar". Its existence is in jeopardy right now in the Canary builds.

However, I have been working hard to reimplement the old taskbar so everyone can continue to use it in Canary right now and subsequently next year's update.

**Reply by influential-eliot** [reply]
Ahh ... so ... How is this done for Windows 11?

( _sorry, this whole discussion was originally about Windows 11, it is why I'm down here chatting away!_ 😅 )

**Comment by pyrates999** [comment]
This needs to be closed as reinstalling EP with a newer release fixes it.

**Comment by wad11656** [comment]
> That is the Windows 10 taskbar, which I referred to as "old taskbar". Its existence is in jeopardy right now in the Canary builds.
> 
> However, I have been working hard to reimplement the old taskbar so everyone can continue to use it in Canary right now and subsequently next year's update.

What? Is this program going to lose the Windows 10 taskbar functionality? I'm planning on installing this for all our company's users and need the Toolbar functionality. So it'd be good to know if this program is going to stop working soon.

**Reply by Amrsatrio** [reply]
Yeah sadly. I will be releasing the restored taskbar soon so that it will continue to exist. But I cannot guarantee the stability after updates and I don't want to be responsible for damages due to the restored taskbar crashing on company PCs, so yeah sad story overall.
